tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -403,19 +403,22 @@ function setupNpmIgnoreConfig(tree: Tree, options: NormalizedSchema) {
 }
 
 function updateNpmScripts(tree: Tree, options: NormalizedSchema) {
+  /* eslint-disable @fluentui/max-len */
+  const scripts = {
```

Comment:
hoisted updated scripts to one common dictionary for better readability and updated `build:local`